Domain/DTO/Contador/ContadorDTO.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace safi_gerenciamento_xml.Domain.DTO.Contador
+{
+    public class ContadorDTO
+    {
+        [Key]
+        public Guid? Id_Contador {get;}
```

Comment:
Verifique na documentação do ORM se já adicionar uma propriedade com o nome "Id" não é o suficiente para considerar como primary key

Reply:
Verifiquei, e o DataStax contêm `Data Annotations` para mapear essas informações como `Clustering Keys` e `Partition Keys`. Eu poderia usar, mas é meio que obrigação de quem usa o DTO saber se é ou não um índice, no caso, quando eu for usar o LINQ pra consulta, eu vou saber já quem é quem.

doc: https://docs.datastax.com/en/latest-csharp-driver-api/api/Cassandra.Mapping.Attributes.html